TS02-Convert-to-Date-Time-Class-POSIX.Rmd

```diff
@@ -86,22 +87,12 @@ library(lubridate)  #work with dates
 #set working directory to ensure R can find the file we wish to import
 #setwd("working-dir-path-here")
 
```

Comment:
I think there is a way to reject this change I made when doing the pull request. But I am not sure how that part works. Maybe Leah can help. Or go ahead with merging as is and then add it back in. Sorry I can't be more helpful there!